### DISCUSSION TITLE
Suggestion to add other refresh rates to choose from

### DISCUSSION THREAD

**Discussion by SwierzewskiP** [BartoszCichecki/LenovoLegionToolkit#1505] [Ideas]
Could you add 80 (and maybe 90 Hz) refresh rates to choose? This way it would be possible to cap framerate on good 40 (45 in case of 90) for perfect frame pacing under VRR window and still preserve battery life, without as high refreshes as 120+.
Honestly, 75 Hz now present in the app seems of no use in this regard, as it don't provide any good fps option. 72 Hz would be more useful with its 36 fps possible cap.

**Comment by BartoszCichecki** [comment]
Hey,
Available refresh rates are reported by Windows and they are displayed in LLT. If you need/want additional ones, you should use a utility like [CRU](https://www.monitortests.com/forum/Thread-Custom-Resolution-Utility-CRU) to add more refresh rates if your panel can support them.

**Comment by SwierzewskiP** [comment]
Hey,

My refresh rates in Windows settings are different than those in LLT. As I
recall there was 60-120 adaptive, 120-240 adaptive, 60, 120, 240. 75 was
not present there, only in LLT…

Best regards,
Piotr

W dniu pon., 2 gru 2024 o 16:36 Bartosz Cichecki ***@***.***>
napisał(a):

**Reply by BartoszCichecki** [reply]
Yes, that is unfortunately a discrepancy in Windows settings. Microsoft uses some sort of new API that supports adaptive refresh rates. This API is not available for 3rd party developers, so LLT just like other apps uses the "old" API. The point still stands. If you configure the refresh rate with something like CRU, it will appear there.
